05/app.js

```diff
@@ -0,0 +1,111 @@
+// konstruktor
+function Person(initialName, initialSurname) {
+    this.name = initialName;
+    this.surname = initialSurname;
+    this.grades = [];
```

Comment:
Rozmawialiśmy, że to ma być obiekt, a nie tablica.

Reply:
Mateusz, jest to zadanie omawiane podczas 2 konsultacji, bez rezultatów jak widać. Teraz te komentarze nie są dla mnie jasne. Mam wrażenie kręcenia się w kółko. Dzisiaj będę to omawiała z Mateuszem, potrzebuję może innej osoby i innych wyjaśnień.